helm/hpcc/values.yaml

```diff
@@ -49,6 +49,75 @@ security:
     extern: "allow"
     datafile: "allow"
 
+## placements
```

Comment:
As a general comment (not specific to this PR but we need to think about it), the values.yaml is starting to get very large. Do we want so much in-line documentation? Is this the best location for the information, and is it starting to make the values.yaml hard to read/follow? Is there a better solution?

Reply:
Another solution is to instruct user:
1) helm template to generate yaml file
2) use kustomize or similar tool to patch whatever changes 
3) kubectl apply/create to deploy the cluster
4) kubectl patch to change dynamic settings

Reply:
I don't object to the values file being the place where we set this information.

I'm questioning whether the default values file is the best place to document this information

Reply:
Maybe just point a document for user and move all instructions including examples to the documentation?. I am OK with it.

Reply:
I think that's sensible, but there are pre-existing sections that could also benefit from moving out into a document.
Let's leave this as it is for now, and tackle tidying up/referencing doc(s) from values.yaml in a separate JIRA/PR

Reply:
Or put placements instructions to a file, such as Placements.md